### ISSUE TITLE
δέ in Rom 1:11, Rom 15:8

### ISSUE THREAD

**Issue by jonathanrobie** [biblicalhumanities/greek-new-testament#8]
Rom 1:11 is misparsed: δέ should join its clause at a higher level than the preceding articular infinitive. Same with Rom 15:8.

**Comment by rkjtan** [comment]
The tree interprets εἱς τὸ στηριχθῆναι ὐμᾶς as being corrected to εἱς τὸ συμπαρακληθῆναι in Rom 1:11. This is also the view of commentators like Jewett in the Hermeneia series: "The verb συμπαρακληθῆναι (“to be mutually encouraged, exhorted”) in the aorist passive stands parallel to the aorist passive infinitive “to be strengthened” in the preceding verse, which means that Paul wishes to qualify the process of apostolic strengthening." Our interpretation of Rom 15:8 is typical, as shown by the following comment in the UBS Handbook: "The TEV understands these words as introducing a second reason why Christ became a servant to the Jews and so introduces the words also to enable to make this relation clear. Most translations follow this same interpretation, though it is possible to connect the first part of verse 9 with the previous verse in a different way."

**Comment by stemmatic** [comment]
The tree has τοῦτο δὲ ἐστιν συμπαρακληθῆναι ... under the scope of τὸ in εἰς τὸ στηριχθῆναι ὑμᾶς. Prepositions don't govern clauses like that. What we have is a parallel independent clause is with some ellipsis. The quoted commentators merely discuss the semantics, not the syntax. Alternatively, one could view τοῦτο δὲ ἐστιν as some sort of parenthetical discourse marker but that does not really take into account the δέ.

**Comment by rkjtan** [comment]
First, let me say that we're not claiming that our trees are always a correct representation of the syntax (for example, from time to time we still find errors/oversights/inconsistencies that slipped through while using the trees for various types of internal research) or that it is the only possible representation (even when it is a technically correct one). So, thank you for your feedback. It is most definitely welcome.

On the one hand, I can grant the possibility of a parallel independent clause with some ellipsis. However, we don't really have a good way to represent such ellipsis with our trees under the current scheme we use.

On the other hand, I'm not sure you can make such a hard & fast distinction between semantics & syntax. Furthermore, τοῦτο δὲ ἐστιν occurs only once in the NT. There are probably extra-biblical examples that could be appealed to for reference--anyone who has found such examples would be most welcome to bring them into the discussion. Moreover, I'm not sure anyone can successfully command universal assent to a sweeping assertion like "Prepositions don't govern clauses like that," without the backing of exhaustive corpus evidence of all infinitival clauses conjoined in some way that are may be governed by the same preposition & in particular of any cases involving τοῦτο δὲ ἐστιν & an infinitival clause in this type of situation. The simplest hypothesis & the simplest representation of the meaning & possible syntax still seems, in my opinion, to be what is currently in the tree.

**Comment by stemmatic** [comment]
I am not proposing an idiosyncratic analysis. On the contrary, these cases all came up because they conflict with the analysis of articular infinitives by Denny Burk in his detailed monograph on these constructions in the New Testament. Given the detail of his study and effort gone into it, I recommend that he and his work be further consulted. He probably has studied the syntax of these particular passages to a greater extent than anyone else at the moment. I don't have a dog in this fight, and I don't have the time to get into a point-by-point argument, but I thought it would be worthwhile to point out analyses that conflict with the state of current scholarship.

**Comment by rkjtan** [comment]
Please don't take me the wrong way. I don't think & never meant to imply in any way that you were proposing an idiosyncratic analysis. Thanks for pointing to Denny Burk's monograph. I haven't seen it. Perhaps I could invite Denny to weigh in, if he's interested & has the time.

**Comment by jonathanrobie** [comment]
For context, Stephen and I are seeing if we can construct a complete list of articular infinitives using queries on the lowfat treebanks.  See here:

http://www.ibiblio.org/bgreek/forum/viewtopic.php?f=11&t=2887&start=70#p19457

We can currently find 290 with a straightforward query, and we are trying to find the missing ~30 cases and eliminate one false positive. To do this, we need to (1) improve the query, and (2) correct any treebank issues.

**Comment by jonathanrobie** [comment]
Let me make sure we're discussing the same thing.  Here is the current representation in the SBL Lowfat tree:

![romans1 11-12](https://cloud.githubusercontent.com/assets/1767869/6618227/81dc1230-c897-11e4-8850-1958c9e08238.png)

I don't have an easy way to show the representation in the GBI version graphically, but it seems to have the same relative hierarchy.

**Comment by rkjtan** [comment]
This is an interesting issue. Thanks for pointing me to it. I had not read the discussion before my original responses to the questions here, which led to a lack of context in understanding where the questions where coming from.

I may have gotten too hung up on explaining why we did this the way we did. We know that we had made many decisions that can be questioned/disputed & we just wanted to let people know what the reasons were. When a tree is demonstrably in error, it will definitely be changed.  When there are multiple defensible interpretations, my preference is to record each & every interpretation. 

For a list of articular infinitives, for the sake of completeness & to more fully meet the needs of researchers coming at the question from different angles, I think that there should be a list of undisputed articular infinitives & also a list of potential (disputed) articular infinitives, together with the reasons why they are disputed. Otherwise, new researchers coming at the question from a fresh angle will always have to do much of the same work of casting a wide net & culling examples again & cannot rely on previous work—even previous work that can be convenient pulled up in a query in a split-second. A good way of adding, evaluating, editing, & querying alternative trees is a very much needed addition to the trees, in my opinion. It is an ongoing task that requires a strong, vibrant community constantly using, improving, & building on top of the trees. I intend to open a new issue on how best to do that.
